### ISSUE TITLE
View is stuck

### ISSUE THREAD

**Issue by heffneil** [home-assistant/iOS#1708]
**iOS device model, version and app version**
Model Name:  iPhone Pro Max
Software Version: 14.6
App version: 2021.6 (2021.158)

**Home Assistant Core Version**
core-2021.6.6
--

When I click on the app, and it appears to have been running on the back, I will click on a top menu bar to change views.  The selection at the top will change however the view below does not change.  If I click on another view link on the top bar other than the first one it doesn't load.  I then click back on the first view and it will switch that option on the top menu bar and then going back to the original selection will work.

In short: the view/menu bar option at the top is changing, the view is not.  It appears "stuck" on the first page.  I have to choose several options to unstick it.  If I manually end the app and reload it - it appears to work as expected.

I would call this a weird caching problem but it's a hunch.  I thought about turning off the Remember Last Page to see if that would fix it. I haven't tried yet.

**Comment by dof250** [comment]
I'm also running in this issue almost every time I launch the Home Assistant app. I always first have to refresh the page to get live data.

**Comment by heffneil** [comment]
Good to know I am not alone.  I don't know how to replicate it but it feels like a stale cache that has to be cleared for whatever reason.

Nothing I can do seems to fix it. If you find a solution please post!

**Comment by gieljnssns** [comment]
I'm in the same boat. 
I have to refresh the app constantly to get live data or toggle some switches, lights or scripts…

**Comment by zacwest** [comment]
The app is likely not in control of this. I recommend taking a look at the frontend support channels such as [Discord](https://www.home-assistant.io/join-chat/), the [forums](https://community.home-assistant.io/), or [GitHub issues](https://github.com/home-assistant/frontend).
